tests/e2e/test_rosacli_cluster.go

```diff
@@ -328,9 +328,149 @@ var _ = Describe("Edit cluster",
 					"-y")
 				Expect(err).To(HaveOccurred())
 				textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
-				Expect(textData).
+        Expect(textData).
 					To(ContainSubstring(
-						"ERR: Schedule '\"5 5\"' is not a valid cron expression"))
+						"ERR: Schedule '\"5 5\"' is not a valid cron expression"))			
+      })
+
+		// Excluded until bug OCM-8408 is resolved
+		It("can verify delete protection on a rosa cluster - [id:73161]",
+			labels.High, labels.Runtime.Day2, labels.Exclude,
+			func() {
+				By("Enable delete protection on the cluster")
+				deleteProtection := "Enabled"
+				_, err := clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=true",
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
```

Comment:
Add a defer step to disable the delete protection. Otherwise any coming check steps failure will cause the cluster teardown issue.